psydac/feec/derivatives.py

```diff
@@ -12,10 +12,9 @@
 from psydac.fem.basic       import FemField, FemSpace
 from psydac.linalg.basic    import LinearOperator
 from psydac.ddm.cart        import DomainDecomposition, CartDecomposition
-
+from psydac.fem.basic       import FemLinearOperator
```

Comment:
In several places below the constructor of FemLinearOperator is used, for clarity I suggest that the arguments are passed with keyword (`linop` or `sparse_matrix`)